api/bucketsd/pb/bucketsd.proto

```diff
@@ -291,7 +298,6 @@ service APIService {
     rpc DefaultArchiveConfig(DefaultArchiveConfigRequest) returns (DefaultArchiveConfigResponse) {}
     rpc SetDefaultArchiveConfig(SetDefaultArchiveConfigRequest) returns (SetDefaultArchiveConfigResponse) {}
     rpc Archive(ArchiveRequest) returns (ArchiveResponse) {}
-    rpc ArchiveStatus(ArchiveStatusRequest) returns (ArchiveStatusResponse) {}
-    rpc ArchiveInfo(ArchiveInfoRequest) returns (ArchiveInfoResponse) {}
+    rpc Archives(ArchivesRequest) returns (ArchivesResponse) {}
```

Comment:
Pointing out that, in theory, this is a breaking change. I'm not sure if Slate uses this or similar so might deserve some preparation for this version release.

Reply:
Yea, I will treat it as a breaking change and do all the necessary things.